firstvoices/backend/search/signals/media_signals.py

```diff
@@ -23,6 +24,16 @@ def remove_audio_from_index(sender, instance, **kwargs):
     request_remove_from_index(AudioDocumentManager, instance)
 
 
+@receiver(post_save, sender=Document)
+def sync_document_in_index(sender, instance, **kwargs):
+    request_sync_in_index(DocumentDocumentManager, instance)
+
+
+@receiver(post_delete, sender=Document)
+def remove_document_from_index(sender, instance, **kwargs):
+    request_remove_from_index(DocumentDocumentManager, instance)
+
+
```

Comment:
Are we missing signals to update the has_document (and has_ other media) values in the DictionaryEntryDocument (and Song and Story) when related media is edited? If so, it could be handled in a separate ticket but we should make sure that is covered

Reply:
I am not sure I understood this, so I may be thinking of something else. But from what I understand is when we update the `DictionaryEntry` through FE or APIs, the `DictionaryEntry` document gets updated, and thus fetches the right count for has_image, has_audio etc. 
Is this what you're referring to ?

Reply:
Yes, just wondering because we have separate signals to handle the m2m through model being updated for categories and related dictionary entries

Reply:
Created a separate ticket: [FW-6404](https://firstvoices.atlassian.net/browse/FW-6404)